### ISSUE TITLE
Duplicate assaytype/assayname endpoints

### ISSUE THREAD

**Issue by shirey** [dbmi-pitt/ubkg#56]
Create new endpoints that exactly duplicate the functionality (exact inputs/outputs) of HuBMAP Search API [GET /assaytype/\<name\>](https://github.com/hubmapconsortium/search-api/blob/db5c5aca6d1f0b0716f7c0ba608a4ca25b158bd3/src/libs/hubmap_endpoints.py#L11) and [POST /assayname](https://github.com/hubmapconsortium/search-api/blob/db5c5aca6d1f0b0716f7c0ba608a4ca25b158bd3/src/libs/hubmap_endpoints.py#L12) endpoints by pulling all information from the SenNet/HuMAP application specific ontologies (instead of the assay_types.yaml file).

Alan is going to produce the Cypher query and Charles the endpoint in UBKG '/assaytype/<name>', methods=['GET'] that returns the same JSON as the same endpoint in search-api produces.

I have a pull request for this work out for Alan. I will then push this out to DEV and PROD for the GET part of this. The POST part is blocked by another Card which will fix OpenAPI code generation for POST and PUT endpoints.

**Comment by AlanSimmons** [comment]
@shirey Reassigning to myself. I am currently writing this endpoint, which is based on a Cypher query that @computationdoc wants to review.

**Comment by AlanSimmons** [comment]
@shirey @ChuckKollar once the ontology api endpoint is written, do we want to replace the code in seach-api that currently reads assay_types.yaml? Is that a task in this issue?

**Comment by AlanSimmons** [comment]
@shirey @ChuckKollar 

The ontology endpoint will return a JSON array. The structure of the JSON is slightly different than the YAML file in that the YAML key (that corresponds to the data_type key in the JSON array) is at the same level as the rest of the data.

For example, for the AF data_type, assay_types.yaml looks like:

```
AF:
    description: Autofluorescence Microscopy
    alt-names: []
    primary: true
    contains-pii: false
    vitessce-hints: []

```
but in the JSON, it looks like:

```
[
  {
    "alt-names": [],
    "contains-pii": false,
    "data_type": "AF",
    "dataset_provider": "HuBMAP IEC",
    "description": "Autofluorescence Microscopy",
    "primary": true,
    "vis-only": false,
    "vitessce-hints": []
  },
...
]
  
```

**Comment by shirey** [comment]
The `GET /assaytype/name` was implemented, `POST /assayname` will be implemented under: #67
